v0.1/core/brainAtlasVersion.schema.json

```diff
@@ -0,0 +1,86 @@
+{
+  "$id": "https://schema.hbp.eu/sand/0.1/core/brainAtlas.schema.json",
```

Comment:
Wrong schema name. Change brainAtlas.schema.json --> brainAtlasVersion.schema.json.

Reply:
yes. forgot that one